### PR TITLE
augeas 1.4.0

### PR DESCRIPTION
**Pull request by dunn**

### CODE REVIEW

File: Library/Formula/augeas.rb [hunk 1]
```diff
@@ -12,7 +12,6 @@ class Augeas < Formula
 
   head do
     url "https://github.com/hercules-team/augeas.git"
-
```
Comment (DomT4) [inline Library/Formula/augeas.rb]: I'm the most anal person in the world about this, but not having an empty line between the `url` and the first `depends_on` drives me up the wall :smile_cat:. Feel free to ignore me though.

Comment (dunn) [inline Library/Formula/augeas.rb]: Ahaha, having the line is what bugs me.

Comment (DomT4) [inline Library/Formula/augeas.rb]: Whitespacing makes me feel happy about reading formulae. It feels more readable. I'll go quietly if others prefer no whitespacing though :wink:.

Comment (dunn) [inline Library/Formula/augeas.rb]: I'll add it back in don't worry

Comment (DomT4) [inline Library/Formula/augeas.rb]: :heart: :heart: :heart:

**Pull request closed without merging**